conanfile.txt

```diff
@@ -0,0 +1,18 @@
+[requires]
+eigen/3.4.0
+opencv/4.5.5
+
+[generators]
+cmake_find_package
+virtualrunenv
```

Comment:
Trenger vel ikke denne generatoren?

Reply:
Vi får se. Den kan brukes for å sette opp blant annet linker path, slik at du finner riktige so-filer (dll-filer) når du skal kjøre programmet. Hvis labene kjører fint uten, så kan vi prøve det.

Reply:
Lar den være med enn så lenge, tenker at den gjør ikke mye skade.